sound/sound-font-config.ini-configuration.md

```diff
@@ -25,111 +27,206 @@ volHum=15
+# a swing effect is triggered. 
+# Default 250.
 ProffieOSSwingSpeedThreshold=250
 
 # How much to bend the response curve between swing speed and swing volume.
-# Can be 0.01 - 2.0, where value of 1.0 will result in no bending. Default 0.5
+# Can be 0.01 - 2.0, where value of 1.0 will result in no bending. 
+# Default 0.5
 ProffieOSSwingVolumeSharpness=0.5
 
 # The volume multiplier when swings are at the swing speed threshold.
-# Can be 1.0 to 3.0. Default 2.0
+# Can be 1.0 to 3.0. 
+# Default 2.0
 ProffieOSMaxSwingVolume=2.0
 
 # Specify what fraction of swing that must be played before a new swing can be
-# started. Can be 0.0-1.0. Defaults to 0.5 (50%).
+# started. Can be 0.0-1.0. 
+# Defaults to 0.5 (50%).
 ProffieOSSwingOverlap=0.5
 
 # This is used to control the volume of the combined hum and smoothswings
-# when an accent swing plays.                       
+# when an accent swing plays.
+# Values range from 0.0 to 1.0, with 0 being no reduction, and 1
+# meaning that when the accent swing is at peak volume, the
+# smooth swing pairs are totally muted.                       
 # Defaults to 0.2 (volume is decreased by 20% of swing volume)    
-ProffieOSSmoothSwingDucking=0.2 
+ProffieOSSmoothSwingDucking=0.2  
 
-# How slow (degrees per second) the swing has to be before it's not considered a
-# swing anymore. Default 200.
+# How slow (degrees per second) the swing has to be before it's not considered
+# to be a swing anymore. 
+# Default 200.
 ProffieOSSwingLowerThreshold=200
 
-# Only used for non-smoothswing fonts. Specifies how aggressive a swing has to be to be considered a slash. Once we
+# Only used for non-smoothswing fonts. Specifies how aggressive a swing has
+# to be to be considered a slash. Once we
 # reach the ProffieOSSwingSpeedThreshold, rate of swing speed change is used to
-# determine if it's a swing or a slash. Default 260
+# determine if it's a swing or a slash.
+# Default 260
 ProffieOSSlashAccelerationThreshold=260
 
-# For OLED displays. This specifies the frame rate of animations.
+# If #define ENABLE_SPINS is defined:
+# Number of degrees the blade must travel while staying above the
+# swing threshold in order to trigger a spin sound.
+# Default is 360 or one full rotation.
+ProffieOSSpinDegrees=360.0
+
+# ---- As of ProffieOS 6 ---- #
+
+# Minimum acceleration for Accent Swing file Selection
+# recommended value is 20.0 ~ 30.0
+ProffieOSMinSwingAcceleration=20.0
+
+# Maximum acceleration for Accent Swing file Selection
+# must be higher than Min value to enable selection
+# recommended value is 100.0 ~ 150.0
+ProffieOSMaxSwingAcceleration=100.0
+
+# Set the volume for each effect individually, in percent.
+# 50 makes it half as loud. 200 makes it twice as loud. 
+# Maximum allowed value is currently 255. The default is 100.
+# EFFECTNAME can be "clash", “preon”, “out”, “pstoff”, etc.
+#
+# ProffieOS.SFX.EFFECTNAME.volume=100
+ProffieOS.SFX.boot.volume=75
```

Comment:
In this file, every variable should be set to it's default value.

Reply:
fixed